src/components/CampaignInitialisation/CampaignInitialisationScreen.tsx

```diff
@@ -105,6 +105,11 @@ export const CampaignInitialisationScreen: FunctionComponent<NavigationProps> =
     if (updateCampaignConfigError) {
       if (updateCampaignConfigError instanceof CampaignConfigError) {
         Sentry.captureException(updateCampaignConfigError);
+        Sentry.addBreadcrumb({
+          data: {
+            operatorToken: authCredentials.operatorToken
+          }
+        });
```

Comment:
haha I suspect we should add the breadcrumb before `captureException` instead haha 😆